comp/otelcol/otlp/components/exporter/logsagentexporter/logs_exporter.go

```diff
@@ -61,17 +63,32 @@ func NewExporterWithGatewayUsage(
 	}
 
 	return &Exporter{
-		set:              set,
-		logsAgentChannel: logsAgentChannel,
-		logSource:        logSource,
-		translator:       translator,
-		gatewaysUsage:    gatewaysUsage,
-		cfg:              cfg,
+		set:                set,
+		logsAgentChannel:   logsAgentChannel,
+		logSource:          logSource,
+		translator:         translator,
+		gatewaysUsage:      gatewaysUsage,
+		orchestratorConfig: cfg.OrchestratorConfig,
+		cfg:                cfg,
 	}, nil
 }
 
-// ConsumeLogs maps logs from OTLP to DD format and ingests them through the exporter channel
+// ConsumeLogs checks the scope of the logs and routes them to the appropriate consumer
 func (e *Exporter) ConsumeLogs(ctx context.Context, ld plog.Logs) (err error) {
+	scope := getLogsScope(ld)
+	switch scope {
+	case K8sObjectsReceiver:
+		if e.orchestratorConfig.Enabled {
```

Comment:
if `e.orchestratorConfig.Enabled` is disabled, any logs should be routed to `e.consumeRegularLogs(ctx, ld)` to match current logs behavior